lms/djangoapps/branding/__init__.py

```diff
@@ -40,7 +41,7 @@ def get_visible_courses(org=None, filter_=None):
     else:
         courses = CourseOverview.get_all_courses(filter_=filter_)
 
-    courses = sorted(courses, key=lambda course: course.number)
+    courses = courses.order_by('id')
```

Comment:
could you clarify why you're making the change from number to id?

Reply:
I added some information to the commit message. I didn't put it as a comment here, because it would have been strange for the comment to reference code that already existed.